File: PostalWeb/PostalWeb/WebContent/service/clientservice.js
/**
 * 
 */
app.factory('ClientService',function($http){
	var clientService={}
	var BASE_URL="http://localhost:8080/PostalWeb"
	
	clientService.login=function(client)
	{
	return $http.post(BASE_URL+"/servlet/login",client)
	}
	clientService.logout=function()
	{
	return $http.get(BASE_URL+"/servlet/logout")
	}
	clientService.getlist=function(){
		return	$http.get(BASE_URL + "/servlet/getlist")
		}
	clientService.getatelist=function(){
		return	$http.get(BASE_URL + "/servlet/getatelist")
		}
	clientService.complete=function(){
		return	$http.get(BASE_URL + "/servlet/complete")
		}
	clientService.update=function(client)
	{
	return $http.put(BASE_URL+"/servlet/update",client)
	}
	/*clientService.bill=function(client)
	{
	return $http.get(BASE_URL+"/servlet/bill",client)
	}*/
	return clientService;
	
})

File: PostalWeb/PostalWeb/src/com/postalweb/service/ClientService.java
package com.postalweb.service;

import java.util.List;

import com.postalweb.model.Client;

public interface ClientService {
	public Client addlead(Client client);
	public Client login(Client client);
	
	public Client leadcount(Client client);
	public List<Client> list();
	public List<Client> atelist();
	public List<Client> complete();
	public Client update(Client client);
	public Client bill(Client client);
	public Client leadprocess(Client client);
	public Client leadverify(Client client);
	public Client leadnotverify(Client client);
}
